pom.xml

```diff
@@ -4,19 +4,19 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>com.exasol</groupId>
     <artifactId>s3-document-files-virtual-schema</artifactId>
-    <version>2.1.1</version>
+    <version>2.1.2</version>
     <name>Virtual Schema for document data in files on AWS S3</name>
     <description>Adapter for document data access from files from AWS S3.</description>
-    <url>https://github.com/exasol/s3-document-files-virtual-schema</url>
+    <url>https://github.com/exasol/s3-document-files-virtual-schema/</url>
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
         <java.version>11</java.version>
         <junit.version>5.8.2</junit.version>
-        <jacoco.version>0.8.7</jacoco.version>
+        <jacoco.version>0.8.8</jacoco.version>
         <gpg.skip>true</gpg.skip>
-        <org.testcontainers.version>1.16.3</org.testcontainers.version>
-        <vs-commom-document-files.version>6.0.1</vs-commom-document-files.version>
+        <org.testcontainers.version>1.17.1</org.testcontainers.version>
+        <vs-commom-document-files.version>7.0.0</vs-commom-document-files.version>
         <!--
                 M4 does not fail tests if beforeAll fails
                 M5 fails in integration tests with a strange error
```

Comment:
Can't comment on the next line, but I think you can also delete the `surefire.and.failsafe.plugin.version` property